File: Data Structure/Stacks/stacks.java
import java.util.Stack;
public class stacks {
    public static void main(String[] args){
        Stack<Integer> s = new Stack<>();
        s.push(1);
        s.push(2);
        s.push(3);
        System.out.printf("Length: %d\n", s.size());
        System.out.printf("Length: %b\n", s.isEmpty());
        while (!s.isEmpty()) {
            System.out.println(s.peek());
            s.pop();
        }
    }
}

File: Data Structures/Stacks/stacks.cpp
#include <iostream>
#include <stack>
using namespace std;

int main(){
    stack<int> s ;
    s.push(1);
    s.push(2);
    s.push(3);
    cout<< s.top();
    s.pop();
    cout<<"length: "<<s.size()<<endl;
    cout<<"empty: "<<s.empty()<<endl;
    while(s.empty()!=1){
        cout<< s.top();
        s.pop();
    }
    return 0;
}